src/utils/autorestOptions.ts

```diff
@@ -199,11 +221,11 @@ async function getPackageDetails(host: Host): Promise<PackageDetails> {
 }
 
 export async function getCredentialScopes(
-  host: Host
+  host: AutorestExtensionHost
 ): Promise<string[] | undefined> {
-  const addCredentials = await host.GetValue("add-credentials");
-  const credentialScopes = await host.GetValue("credential-scopes");
-  const azureArm = await host.GetValue("azure-arm");
+  const addCredentials = await host.getValue("add-credentials");
```

Comment:
Maybe that's for a follow up PR if you care about it but the latest version just expose the configuration in the session object with `.configuration` so you can get it without await everywhere
```ts
session.configuration["add-credentials"]
```